RecoTracker/MkFitCore/src/PropagationMPlex.cc

```diff
@@ -1,3 +1,5 @@
+#include "FWCore/Utilities/interface/CMSUnrollLoop.h"
```

Comment:
please use ifdef to keep the standalone version working

Reply:
I was planning to import it into the standalone version, either through mkfit-externals or a download in the configure script, so we can do performance tests of unrolling/vectorization in the standalone version.

Reply:
Hi @dan131riley are you planning to update the standalone part in this PR?